omg/entities/player.py

```diff
@@ -58,7 +57,7 @@ def __init__(self, name, char_class, image_file, scale, initial_angle=0):
         self.mana_regen_cooldown = 0
 
         # Skills
-        self.element = SkillManager(N_SKILLS_MAX)
+        self.element = ElementManager(N_ELEMENTS_MAX)
```

Comment:
I am still in favor of `self.elements` instead of `self.element` as this is an array of elements.